.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,36 @@
+# Summary
+
+Please provide a summary of this issue.
+
+## If You Are Reporting a Bug
+
+If you're reporting a bug, please provide a detailed description of the problem.
+
+### Replication Stps
```

Comment:
Fixed in https://github.com/deis/builder/pull/233/commits/c9e4cdef8eeabe2390d089c8cdbed4c53dfc0024